easybuild/easyconfigs/a/AUGUSTUS/AUGUSTUS-3.4.0-foss-2021a.eb

```diff
@@ -0,0 +1,64 @@
+# Updated by: Sebastien Moretti (SIB)
+
+easyblock = 'ConfigureMake'
+
+name = 'AUGUSTUS'
+version = '3.4.0'
+
+homepage = 'https://bioinf.uni-greifswald.de/augustus/'
+description = "AUGUSTUS is a program that predicts genes in eukaryotic genomic sequences"
+# software_license = Artistic
```

Comment:
consider adding a corresponding license entry to: https://github.com/easybuilders/easybuild-framework/blob/develop/easybuild/framework/easyconfig/licenses.py

```suggestion
# software_license = 'LicenseArtisticv1'
```

Reply:
done